src/pages/EventAdministration/components/EventFileSender.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import { useSendFilesToAttendees } from 'hooks/Event';
+import { useSnackbar } from 'hooks/Snackbar';
+import { FileUploader } from 'react-drag-drop-files';
+
+// Material-UI
+import { Button, IconButton, List, ListItem, ListItemIcon, ListItemText, Box, Typography } from '@mui/material';
+import { AttachFileRounded, CloseRounded } from '@mui/icons-material/';
+
+// Project components
+import Dialog from 'components/layout/Dialog';
+
+export type EventFileSenderProps = {
+  eventId: number;
+};
+
+const EventFileSender = ({ eventId }: EventFileSenderProps) => {
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const showSnackbar = useSnackbar();
+  const [files, setFiles] = useState<File[]>([]);
+  const fileTypes = ['JPG', 'PNG', 'GIF', 'PDF', 'DOCX'];
+
+  const handleFileUpload = (newFiles: FileList) => {
+    setFiles([...Array.from(newFiles).filter((newFile) => files.every((file) => file.name !== newFile.name)), ...files]);
```

Comment:
Her sammenligner jeg to filnavn og sjekker om de er like. Det hadde vært mer optimalt å sammenligne de faktiske filene eller hash verdien til filene. Noen forslag på hvordan det gjøres? 

Ellers tror jeg det ikke er sååå viktig ettersom filene bør ha ulike navn fra starten av, men det kan sikkert forårsake litt frustrasjon hvis det ikke går.

Reply:
Tenker at du ikke trenger å ta hensyn til om en annen fil allerede har det samme navnet, det er opp til bruker å passe på det. Kan eventuelt vise en snackbar om det er tilfelle, men bør nok ikke filtrer ut filer med samme navn, de kan jo fremdeles ha forskjellig innhold
```suggestion
    setFiles((prev) => [...prev, ...files]);
```